frontend/ui/widget/pagebrowserwidget.lua

```diff
@@ -1626,6 +1626,17 @@ function PageBrowserWidget:onTap(arg, ges)
     return true
 end
 
+local function openBookMap(self, page)
+    local extra_symbols_pages = {}
```

Comment:
If you provide/expect `self`, it may as well be a real PageBrowserWidget:openBookMap(page) method (like you initially did).